include/Server.hpp

```diff
@@ -52,8 +53,8 @@ class Server
 	private:
 		void		_bindNewSocketToPort(char *port);
 		void		_addFdToEpoll(int new_fd) const;
-		int			_handle_message(epoll_event &event);
-		void		_handleReadyFds(int event_count, struct epoll_event *events);
+		int			_handle_message(epoll_event &event, Configuration &config);
+		void		_handleReadyFds(int event_count, struct epoll_event *events, Configuration &config);
```

Comment:
These parameters aren't necessary if the Configuration is stored as an attribute